int64 degrees
---
string response
